common/components/WalletDecrypt/components/WalletButton.tsx

```diff
@@ -86,24 +86,47 @@ export class WalletButton extends React.PureComponent<Props> {
       >
         <div className="WalletButton-inner">
           <div className="WalletButton-title">
-            {icon && <img className="WalletButton-title-icon" src={icon} />}
+            {icon && <img className="WalletButton-title-icon" src={icon} alt="Logo" />}
             <span>{name}</span>
           </div>
 
-          {description && <div className="WalletButton-description">{description}</div>}
-          {example && <div className="WalletButton-example">{example}</div>}
+          {description && (
+            <div
+              className="WalletButton-description"
+              aria-label="description"
+              aria-hidden={!isSecure}
```

Comment:
Why would this be aria hidden if it's insecure?

Reply:
I found the descriptions on the insecure wallet types to a bit annoying when using a screen reader. For example, the private key description is just a private key. Someone using a screen-reader would just head a series of numbers and letters, and have to piece together it's an example of a private key. 

I think think what you're getting at is that this could be handled better, and I agree. It would make more sense to add an aria-label describing the example private key instead of hiding it or providing it to the user with no context.

Reply:
See above, just remove `aria-hidden` attribute.